### PR TITLE
Feature/twitter api call

### PR DESCRIPTION
**Pull request by cantunca**
Implemented an awesome feature

### CODE REVIEW

File: second.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
-Something done more beautifully
+Something done more elegantly
```
Comment (cihatbaktir) [inline second.txt:1]: Yes, this feature seems nice.

File: second.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
 Something else
 I am the master
+
+Implemented an awesome twitter api call.
```
Comment (cihatbaktir) [inline second.txt]: This feature has a minor bug, fix it before merging.

File: second.txt [hunk 3]
```diff
@@ -1,6 +1,5 @@
-Something done more beautifully
+Something done more elegantly
 Something else
-I am the master
-No, I am the master, obey your master!
-No, you both are not masters, kneel down to your actual master!
```
Comment (cantunca) [inline second.txt:5]: @mericturan In your face

**Review by cantunca — COMMENTED** [review]
Muazzam bir feature

**Pull request merged**